src/dataDisplay/tableHeaderCell.jsx

```diff
@@ -1,32 +1,360 @@
-'use strict';
-
-import React, { Component } from 'react';
-import _ from 'lodash';
 import ClassNames from 'classnames';
+import _ from 'lodash';
```

Comment:
What?  A blanket Lodash import from Oscar? 😁

Reply:
This was an existent error :P lodash function were already there (like _.find, _.isNumber, etc) but lodash was not imported, I just imported it and kept it that way so I'm investing time testing things out of the scope.

Reply:
Fair enough 😁

Reply:
Resolving this thread.

I am hoping that we can someday soon do an ESLINT and cleanup pass on React CM UI.  I feel we could get React CM UI into full compliance with ESLINT and all of our other recommended practices with relatively little effort, and then have build automation run ESLINT and enforce that it stays that way.  The same cannot of course be said for Healthy Church (`church-management` repo) ... :-(